husky_extensions/itembox.js

```diff
@@ -327,7 +327,7 @@ define(function() {
                 // reset items visible when new content is loaded
                 this.viewAll = false;
 
-                if (!!data) {
+                if (!!data && data.length > 0) {
```

Comment:
I hope the media-selection is working anyway? And can you link the issue here, for reference?